rdf-definitions/attributes/sdmx-obs-status.json

```diff
@@ -0,0 +1,112 @@
+{
+    "@context": [
+        "http://www.w3.org/ns/csvw",
+        {
+            "@language": "en",
+            "@base": "https://purl.org/csv-cubed/resources/attributes/sdmx-obs-status"
+        }
+    ],
+    "url": "sdmx-obs-status.csv",
+    "@type": [
+        "http://www.w3.org/2004/02/skos/core#ConceptScheme",
+        "http://www.w3.org/2000/01/rdf-schema#Resource",
+        "http://www.w3.org/ns/dcat#Resource",
+        "http://www.w3.org/ns/prov#Entity",
+        "http://www.w3.org/ns/dcat#Dataset"
+    ],
+    "http://purl.org/dc/terms/identifier": [
+        {
+            "@value": "SDMX Code list for Observation Statuses"
+        }
+    ],
+    "http://purl.org/dc/terms/title": [
+        {
+            "@language": "en",
+            "@value": "SDMX Code list for Observation Statuses"
+        }
+    ],
+    "http://www.w3.org/2000/01/rdf-schema#comment": [
+        {
+            "@language": "en",
+            "@value": "A code list to mark up statistical observations, missing, imputed, modelled, or otherwise usually footnoted"
+        }
+    ],
+    "http://purl.org/dc/terms/description": [
+        {
+            "@type": "https://www.w3.org/ns/iana/media-types/text/markdown#Resource",
+            "@value": "This code list provides coded information about the \"status\" of an observation (with respect to events such as the ones reflected in the codes composing the code list). The observation status provides information on 1) the quality of a value or 2) unusual or missing values. The codes in this list are known to be heterogeneous, that is they represent different concepts. They are put together in one code list for practical and historical implementation reasons."
+        }
+    ],
+    "http://purl.org/dc/terms/license": [
+        {
+            "@id": "Copyright 2019, All Rights Reserved"

```

Comment:
Is this really a URI? Didn't we come up with a URI for this at some point?

Reply:
https://choosealicense.com/no-permission/

https://github.com/GSS-Cogs/reusable-rdf-resources/blob/main/licenses.json#L12